doc/api/process.md

```diff
@@ -1754,5 +1754,5 @@ cases:
 [Readable]: stream.html
 [Child Process]: child_process.html
 [Cluster]: cluster.html
-[`process.exitCode`]: #processexitcode-1
+[`process.exitCode`]: #process_process_exitcode
```

Comment:
@lpinca That’s the anchor for `process.exit(code)`, though, not for `process.exitCode`?